lilypad/server/services/device_codes.py

```diff
@@ -0,0 +1,46 @@
+"""The `DeviceCodeService` class for device_codes."""
+
+from typing import Annotated
+
+from fastapi import Depends, HTTPException, status
+from sqlmodel import Session, select
+
+from ..db import get_session
+from ..models import DeviceCodeTable
+
+
+class DeviceCodeService:
```

Comment:
is there a reason this doesn't use the base service?

Reply:
ok, that makes sense.

we should probably make a new `UnauthenticatedBaseService` or something in the future should the need arise for other services that can't have it